Week3/Day2/README.md

```diff
@@ -0,0 +1,16 @@
+## ETL Design Assignment 2.
```

Comment:
```suggestion
## ETL Design Assignment 3.
```

Reply:
Dai actually this is `Day2/Readme.md` and correct. I updated mistakes in `Day3/Readme.md`. [HERE](https://github.com/Saphall/Leapfrog_Data-Engineering_Assignments/commit/7b0310ff554d473aac500ddfc8f32b72bde364c8#diff-9060e8fa0c9c0320990a5cd88852009d878e64cd91917bf706f6550241a011f6) already.
This is  PR for `Day 3` branch . [HERE](https://github.com/Saphall/Leapfrog_Data-Engineering_Assignments/tree/Day3_Assignment/Week3/Day3).  

Actually the branch creation was problem and I will resolve the conflict.